guillotina/fields/patch.py

```diff
@@ -41,10 +42,7 @@ def field_converter(field, value, context):
         if operation is None:
             raise ValueDeserializationError(
                 field, value, f'"{operation_name}" not a valid operation')
-        if 'value' not in value:
-            raise ValueDeserializationError(
-                field, value, f'Mising value')
-        value = operation(context, value['value'])
+        value = operation(context, value.get('value'))
```

Comment:
I'm a bit concerned about possible side-affects with defaulting to `None` here where maybe it should throw an error.